### ISSUE TITLE
HCAO release 1.0.41

### ISSUE THREAD

**Issue by zoependlington** [ebi-ait/hca-ebi-dev-team#494]
The [HCA Ontology release 1.0.40](https://github.com/ebi-ait/ontology/releases/tag/1.0.40) is out and and needs to be picked up. Tag should also be available automatically on Quay.io.

Could the OLS instance/EBI environment be reloaded and the validator cache cleared please?

Thank you!

Tagging @pnejad, @rachadele @willrockout and @bvarner-ebi to keep in the loop.

**Comment by Wkt8** [comment]
Hi! Please tag one of @Wkt8 and @idazucchi and @ESapenaVentura in the future too, thanks!

**Comment by zoependlington** [comment]
Added to our documentation! 😄

**Comment by gabsie** [comment]
@MightyAx can get this up today

**Comment by MightyAx** [comment]
Release Progress:

- 🔁 Dev: Deployed, Integration Tests Running
- 🔜 Staging: Pending
- 🔜 Production: Pending

**Comment by MightyAx** [comment]
@ESapenaVentura is checking this release.

**Comment by ESapenaVentura** [comment]
I am really unsure how to check for this, I have checked everywhere but there is no tag for the latest image loaded (At least, visible for the user)

I have asked Bradley in the meantime.

What makes me think there has not been any update, is the `loaded` property in here: https://ontology.archive.data.humancellatlas.org/ontologies

**Comment by zoependlington** [comment]
To check if the release is the latest release the version date for HCAO should be the date of the release ticket. e.g. The 1.0.40 release was released on 19th July. Currently the OLS instance is stating 22nd June for both Version IRI and Version, which means it's pointing at the 1.0.38 release. There have since been two releases, 1.0.39 on the 23rd June and then 1.0.40 on the 19th July. The extra June releases were due to a colleague fixing some issues and requiring patch releases while I was on annual leave. The releases can be seen here: https://github.com/ebi-ait/ontology/releases. Perhaps the extra releases have caused issues with the latest release being loaded?

**Comment by ke4** [comment]
I think the `1.0.40` release (tag) is pointing to exactly the same commit hash as the `1.0.39` release. I assume this is where the issue lies.

**Comment by ke4** [comment]
There is a new (fixed) release is out: `1.0.41`

**Comment by ke4** [comment]
The new fixed version of ontology `1.0.41` has been released to all of our environments:
- `dev`
- `staging`
- `prod`

```
Ontology info
Ontology IRI: http://ontology.data.humancellatlas.org/ontologies/hcao
Version IRI: http://ontology.data.humancellatlas.org/ontologies/hcao/releases/2022-08-09/hcao.owl
Ontology id: hcao
Version: 2022-08-09
Number of terms: 22065
Last loaded: Tue Aug 09 14:11:02 GMT 2022
```
